compiler/damlc/lib/DA/Cli/Damlc/Test.hs

```diff
@@ -72,25 +77,71 @@ testRun h inFiles lfVersion coverage color mbJUnitOutput  = do
     deps <- runActionSync h $ mapM getDependencies inFiles
     let files = nubOrd $ concat $ inFiles : catMaybes deps
 
+    -- get all external dependencies
+    extPkgs <- fmap (nubSortOn LF.extPackageId . concat) $ runActionSync h $
+      Shake.forP files $ \file -> getExternalPackages file
+    let extDalfs =
+                [ (Just pId, dalf)
+                | pkg <- extPkgs
+                , let dalfs = NM.elems $ LF.packageModules $ LF.extPackagePkg pkg
```

Comment:
```
            , let modules = NM.elems $ LF.packageModules $ LF.extPackagePkg pkg
```
or am I missing something here? Same for the other names here, you seem to return a list of `(pkgId, module)`

Reply:
it's already weird that the function `dalfForScenario` returns a module and not a dalf. I changed it to `moduleForScenario` and did a `s/dalf/module/`.